src/main/java/pages/FlipkartHomePage.java

```diff
@@ -0,0 +1,29 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class FlipkartHomePage {
+    WebDriver driver;
+    By searchBox = By.xpath("//*[@class='_3704LK']");
+    By popup = By.xpath("/html/body/div[2]/div/div/button");
+    By submit = By.xpath("//*[@class='L0Z3Pu']");
+
+    public FlipkartHomePage(WebDriver driver) {
+        this.driver = driver;
+    }
+
+    public WebElement popup() {
+        return driver.findElement(popup);
+    }
+
+    public WebElement searchBox() {
+        return driver.findElement(searchBox);
+    }
+
+    public WebElement submit() {
+        return driver.findElement(submit);
+    }
```

Comment:
POM dosen't need to have getters for your elements which you have defined. It should rather have method actions like : 

`public void clickSubmit(){
 driver.findElement(submit).click()
}`